proto/server/configEvtHandler.go

```diff
@@ -498,7 +498,7 @@ func updateSmfSelectionProviosionedData(snssai *models.Snssai, mcc, mnc, dnn, im
 			},
 		},
 	}
-	smfSelecDataBsonA := toBsonM(smfSelData)
+	smfSelecDataBsonA := configmodels.ToBsonM(smfSelData)
```

Comment:
We can remove unused  ` func toBsonM(data interface{}) (ret bson.M)`  and
`func mapToByte(data map[string]interface{}) (ret []byte)` functions in this file.